app/assets/stylesheets/homepage.css.scss

```diff
@@ -51,25 +51,47 @@
     padding: 50px;
 }
 
-// .destination-col {
-//     padding: 15%;
-//     margin-left: 20%;
-// }
 
-// .activity-col {
-//     padding: 15%;
-//     margin-left: 20%;
-//     margin-top: 3.5%;
-// }
-
-.destination-col a{
+#homepage-icons a {
   text-decoration: none;
-
 }
 
-.activity-col a{
-  text-decoration: none;
+.icons {
+  margin-left:100px;
+}
 
+.destination-text {
+  font-size: 2.5em;
+  margin-left:70px;
+  margin-top:-10px;
+  color: white;
+  text-shadow:
+  0.05em  .05em .05em black,
+  .01em -.01em .01em black,
+  -.01em  .01em .01em black,
+  -.01em -.01em .01em black;
+}
+.activity-text {
+  font-size: 2.5em;
```

Comment:
Properties should be ordered color, font-size, margin-left, margin-top, text-shadow